home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1791,6 +1916,66 @@ components:
           maxLength: 13
           type: string
           example: "+380966617305"
+    CooperationDataScope:
+      type: string
+      enum:
+        - ALL
+        - ADRESS
+        - APARTMETS
+        - OWNERS
+    TaskStatus:
+      type: string
+      enum:
+        - PENDING
+        - ERROR
+        - PROCESSING
+        - ACCEPTED
+        - DECLINED
+    TaskType:
+      type: string
+      enum:
+        - IMPORT
+        - EXPORT
+    ReadTask:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+          example: 1
+        type:
+          $ref: '#/components/schemas/TaskType'
+        status:
+          $ref: '#/components/schemas/TaskStatus'
+    ReadReport:
```

Comment:
If we will decide to move on Summary naming then I suggest the name for it - 
`ReadTaskSummary`

And again we need to have hierarchy in order to differentiate one type of summer from another.